src/docs/devices/KinCony-AG8/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: KinCony-AG8 (ESP32-S3 Smart IR Controller)
+date-published: 2024-10-12
+
+type: misc
+standard: global
+board: esp32
+---
+
+![Product](AG8-3.jpg "Product Image")
+![Product](AG8_Smart_Controller_diagram.jpg "Product Image")
+
+## Resources
+
+- [ESP32 pin define details](https://www.kincony.com/forum/showthread.php?tid=5888)
+- [YouTube video tour](https://youtu.be/wWhbA8ZlyUA)
+
+## ESPHome Configuration
+
+Here is an example YAML configuration for the KinCony-AG8 board.
+
+```yaml
+esphome:
+  name: ag8
+  friendly_name: ag8
+  platformio_options:
+    board_build.flash_mode: dio
+
+esp32:
+  board: esp32-s3-devkitc-1
+  framework:
+    type: esp-idf
+    sdkconfig_options:
+      SOC_RMT_SUPPORT_RX_PINGPONG: "n"
+# Enable logging
+logger:
+  hardware_uart: USB_SERIAL_JTAG
+# Enable Home Assistant API
+api:
+  encryption:
+    key: "dolIFOGZiGF8GT9wSJVBuoNzVQSHanaieBBbRcj1t48="

```

Comment:
Please remove this value from the file. There should be no passwords or secrets references in any examples. The only exceptions are !secret wifi_ssid and !secret wifi_password.